core/hw/holly/sb.cpp

```diff
@@ -783,7 +783,7 @@ void sb_Init()
 	maple_Init();
 	aica_sb_Init();
 
-#if DC_PLATFORM == DC_PLATFORM_DREAMCAST
+#if DC_PLATFORM == DC_PLATFORM_DREAMCAST && defined(ENABLE_MODEM)
 	ModemInit();
```

Comment:
I’d rather us `#define ENABLE_MODEM 0/1`, using `defined(...)` often leads to typos